sdk/eventhub/Azure.Messaging.EventHubs/tests/Azure.Messaging.EventHubs.Tests.csproj

```diff
@@ -37,4 +37,7 @@
   
   <!-- Import the references to the Azure client libraries -->
   <Import Project="$(MSBuildThisFileDirectory)..\Azure.Client.Library.References.props" />
+
```

Comment:
This file appears to be very out of date.   These changes would cause issues and seem orthogonal to the stated purpose of the PR.   Please revert.